lib/client/src/api/promises/NetworkApi.js

```diff
@@ -0,0 +1,205 @@
+     * @param {module:model/NetworkRequest} networkRequest 
+     * @return {Promise} a {@link https://www.promisejs.org/|Promise}, with an object containing data of type {@link module:model/NetworkOptionsResponse} and HTTP response
+     */
+    networkOptionsWithHttpInfo(networkRequest) {
+      let postBody = networkRequest;
+      // verify the required parameter 'networkRequest' is set
+      if (networkRequest === undefined || networkRequest === null) {
+        throw new Error("Missing the required parameter 'networkRequest' when calling networkOptions");
+      }
+
+      let pathParams = {
+      };
+      let queryParams = {
+      };
+      let headerParams = {
+      };
+      let formParams = {
+      };
+
+      let authNames = [];
+      let contentTypes = ['application/json'];
+      let accepts = ['application/json'];
+      let returnType = NetworkOptionsResponse;
+      return this.apiClient.callApiPromised(
+        '/network/options', 'POST',
+        pathParams, queryParams, headerParams, formParams, postBody,
+        authNames, contentTypes, accepts, returnType, null
+      );
+    }
+
+    /**
+     * Get Network Options
+     * This endpoint returns the version information and allowed network-specific types for a NetworkIdentifier. Any NetworkIdentifier returned by /network/list should be accessible here.  Because options are retrievable in the context of a NetworkIdentifier, it is possible to define unique options for each network.
+     * @param {module:model/NetworkRequest} networkRequest 
+     * @return {Promise} a {@link https://www.promisejs.org/|Promise}, with data of type {@link module:model/NetworkOptionsResponse}
+     */
+    networkOptions(networkRequest) {
+      return this.networkOptionsWithHttpInfo(networkRequest)
+        .then(function(response_and_data) {
+          return response_and_data.data;
+        });
+    }
+
+
+    /**
+     * Get Network Status
+     * This endpoint returns the current status of the network requested. Any NetworkIdentifier returned by /network/list should be accessible here.
+     * @param {module:model/NetworkRequest} networkRequest 
+     * @return {Promise} a {@link https://www.promisejs.org/|Promise}, with an object containing data of type {@link module:model/NetworkStatusResponse} and HTTP response
+     */
+    networkStatusWithHttpInfo(networkRequest) {
+      let postBody = networkRequest;
+      // verify the required parameter 'networkRequest' is set
+      if (networkRequest === undefined || networkRequest === null) {
+        throw new Error("Missing the required parameter 'networkRequest' when calling networkStatus");
+      }
+
+      let pathParams = {
+      };
+      let queryParams = {
+      };
+      let headerParams = {
+      };
+      let formParams = {
+      };
+
+      let authNames = [];
+      let contentTypes = ['application/json'];
+      let accepts = ['application/json'];
+      let returnType = NetworkStatusResponse;
```

Comment:
Is this code generated? Those orphaned param objects look somewhat off :-)

Reply:
Yes, this code is autogenerated by the OpenAPI 3.0 generator.